packages/foreman/rubygem-mail/rubygem-mail.spec

```diff
@@ -16,6 +16,10 @@ BuildRequires: rubygems-devel
 BuildArch: noarch
 # end specfile generated dependencies
 
+# Prefer to consume net-smtp/net-imap/net-pop as a default gem
+Requires: ruby-default-gems
```

Comment:
I think this should have a `< 3` to make sure they really are there.

Reply:
just mimicking what we have in `nokogiri.spec` ;)

https://github.com/theforeman/foreman-packaging/commit/0145c819e4f043896d9c6cf3e4b6733b6cd07a06

Reply:
That's because at least Ruby 3.1 still has racc in `ruby-default-gems` (and pretty sure 3.2 didn't drop it either).

Reply:
Watch me cursing at Ruby, Packaging, and moving things around. (Updated)

Reply:
It wouldn't be the first cursing in this area. I know I did my share.